src/main/scala/uk/gov/nationalarchives/aws/utils/ECRUtils.scala

```diff
@@ -0,0 +1,34 @@
+package uk.gov.nationalarchives.aws.utils
+
+import software.amazon.awssdk.services.ecr.{EcrAsyncClient, EcrClient}
```

Comment:
Unnecessary import: `EcrClient`

Reply:
I've cleared this one and the one below. I used the optimise imports on it so it's tidied up a lot of the awsskd imports.